src/main/java/de/tum/in/www1/artemis/web/websocket/team/TeamWebsocketService.java

```diff
@@ -44,6 +49,9 @@ public TeamWebsocketService(WebsocketMessagingService websocketMessagingService)
     public void sendTeamAssignmentUpdate(Exercise exercise, @Nullable Team existingTeam, @Nullable Team updatedTeam, List<StudentParticipation> participationsOfUpdatedTeam) {
         // Users in the existing team that are no longer in the updated team were unassigned => inform them
         if (existingTeam != null) {
+            if (!Hibernate.isInitialized(existingTeam.getStudents())) {
+                existingTeam = teamRepository.findWithStudentsByIdElseThrow(existingTeam.getId());
+            }
```

Comment:
are there cases where the relation might be already loaded?

Reply:
I checked all paths and can't find any. Removed it.